api/v1alpha1/oadp_types.go

```diff
@@ -326,16 +524,15 @@ func (veleroConfig *VeleroConfig) HasFeatureFlag(flag string) bool {
 	return false
 }
 
-func init() {
-	SchemeBuilder.Register(&DataProtectionApplication{}, &DataProtectionApplicationList{}, &CloudStorage{}, &CloudStorageList{})
-}
-
 // AutoCorrect is a collection of auto-correction functions for the DPA CR
 // These auto corrects are in-memory only and do not persist to the CR
 // There should not be another place where these auto-corrects are done
 func (dpa *DataProtectionApplication) AutoCorrect() {
+	// TODO error instead of changing user object?
```

Comment:
@shawn-hurley I think you once said we should not change values. Should we change this to be in validator or it is ok?